CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ v2.2.0 [unreleased]
 ### Refactoring
 - [#391](https://github.com/latera/homs/pull/391) Remove side menu with tasks.
 
+### Bugfixes
+- [#400](https://github.com/latera/homs/pull/400) Update obsolete dependencies.
```

Comment:
We don't mention such things in changelog :)

Reply:
Okay.jpg